### PR TITLE
Dockerize dc campaign finance watch UI

### PR DESCRIPTION
**Pull request by romoy**
## Description
Allow an individual to docker-compose up and launch a dev environment.

## Motivation and Context
Simplify individual on boarding

## How Has This Been Tested?
Yes,  docker-compose up; updating code and see hot reload of change in browser.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I've talked through the changes with a team lead
- [ ] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Create app directory
+RUN mkdir -p /usr/src/dc-campaign-finance-watch
+WORKDIR /usr/src/dc-campaign-finance-watch
+
+# Install app dependencies
+COPY package.json /usr/src/dc-campaign-finance-watch/
+RUN npm install
+
+# Bundle app source
+COPY . /usr/src/dc-campaign-finance-watch
```
Comment (mkalish) [inline Dockerfile]: Is this just moving the install packages and src code to the working directory?

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "release:patch": "npm version patch && npm publish && git push --follow-tags",
     "serve": "node server.js --env=dev",
     "serve:dist": "node server.js --env=dist",
```
Comment (mkalish) [inline package.json]: Can we adjust the names of these commands slightly to be more obvious which one will use the AWS api and which one will use a local one?

File: Dockerfile (hunk 1 above)
Comment (romoy) [inline Dockerfile]: yea, this docker file was the suggest for node dev https://nodejs.org/en/docs/guides/nodejs-docker-webapp/ 
I'm going to check for prod suggestions.

Comment (romoy) [inline Dockerfile]: It's copying the js files from the current repo dir into the docker image so when the server starts it can use the appropriate files.

**Pull request merged**